docs/src/modules/sandbox/CreateReactApp.ts

```diff
@@ -0,0 +1,90 @@
+export const getHtml = ({ title, language }: { title: string; language: string }) => {
+  return `<!DOCTYPE html>
+<html lang="${language}">
+  <head>
+    <title>${title}</title>
+    <!-- Fonts to support Material Design -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <!-- Icons to support Material Design -->
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+  </head>
+  <body>
+    <div id="root"></div>
+  </body>
+</html>`;
+};
+
+export const getRootIndex = (product?: 'joy-ui' | 'base') => {
```

Comment:
👍

Reply:
This means that we cannot have a Material/Joy related demo on base, we should keep this in mind.